sale_timesheet_task_exclude/README.rst

```diff
@@ -0,0 +1,94 @@
+=============================================
+Sales Timesheet: exclude Task from Sale Order
+=============================================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fsale--workflow-lightgray.png?logo=github
+    :target: https://github.com/OCA/sale-workflow/tree/12.0/sale_timesheet_task_exclude
+    :alt: OCA/sale-workflow
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_timesheet_task_exclude
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/167/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+Allows to exclude specific *Task* and all *Timesheets* tracked towards it from
+Sale Order.
+
+This functionality is not available in Odoo, reported in `odoo/odoo#31042 <https://github.com/odoo/odoo/pull/31042>`_.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+This feature proves itself useful for *By Employee* billing approach, when
+a any timesheet entry tracked towards a specific task need to be excluded
+from the Sale Order.
+
+# Go to *Project > All Tasks*
+# Open specific task for editing
+# Check *Exclude From Billing*
+
+Known issues / Roadmap
+======================
+
+Important: The use of this module makes that the hours of timesheets in the
```

Comment:
This should be put in a bullet list (although only one point now).

Reply:
Fixed